include/CppUTest/CppUTestConfig.h

```diff
@@ -201,6 +201,18 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Handling of systems with a different int-width (e.g. 16 bit).
+ */
+#if CPPUTEST_USE_STD_C_LIB
+#if (INT_MAX == 0x7fff)
+#define CPPUTEST_INT_BIT 2
```

Comment:
How about calling this CPPUTEST_INT_SIZE ? Bit is a bit confusing.

Can you also add a test that does sizeof(int) == CPPUTEST_INT_SIZE as a sanity check?